src/index.html

```diff
@@ -9,6 +9,74 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="logo">
+        <a
+          href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="is-active nav__link"
+              href="#apple">
+              Apple
+            </a>
```

Comment:
please check the spaces all over the document
```suggestion
            <a
              class="is-active nav__link"
              href="#apple"
            >
              Apple
            </a>
```